### ISSUE TITLE
Handle forwarder not running when try to add chart

### ISSUE THREAD

**Issue by jchap-pnnl** [VOLTTRON/volttron#625]
For a remote platform, if you try to add a chart in VC, the chart can't be added if the forwarder isn't running. The status message you get says "Unable to load chart: the historian agent is not available." That would make sense for the local instance, but for remote instances, the message should be that the forwarder isn't available.

**Comment by jchap-pnnl** [comment]
In commit (5093af3), messaging was changed to specify that the VOLTTRON Central platform's historian has to be running to add or update charts. 

For now, if the remote platform's forwarder isn't running, the chart can still be added -- the info just won't update. And the user can look at the status of the forwarder agent in the tree or platforms manager to see if it's running.

But we may want to add a check later to see if the forwarder is running and indicate in the chart if it's not.

**Comment by jchap-pnnl** [comment]
In commit (3d42bd9), an issue was fixed where if the forwarder wasn't running, if you check the box to add a chart, the spinning indicator would never quit spinning and the chart wouldn't add. The cause was that in the historian.query response, the result didn't have a "values" property if the forwarder wasn't running. So I added code to check if the values property is in the result, and if not, checks to see if the forwarder is running. If that's the problem, a status message appears stating that's the problem. And whether that's the problem or something else, either way, now the spinning circle stops and the box is unchecked.
